Lib/test/test_asyncio/test_unix_events.py

```diff
@@ -372,6 +372,127 @@ def test_create_unix_connection_ssl_noserverhost(self):
 
             self.loop.run_until_complete(coro)
 
+    def test_create_unix_datagram_server_existing_path_sock(self):
+        with test_utils.unix_socket_path() as path:
+            sock = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
+            sock.bind(path)
+            sock.close()
+
+            coro = self.loop.create_unix_datagram_server(mock.MagicMock, path)
+            transport, protocol = self.loop.run_until_complete(coro)
+            transport.close()
+            self.loop.run_until_complete(transport.wait_closed())
+            assert protocol.connection_made.call_count == 1
+
+    @unittest.skipUnless(hasattr(os, 'fspath'), 'no os.fspath')
+    def test_create_unix_datagram_server_pathlib(self):
+        with test_utils.unix_socket_path() as path:
+            path = pathlib.Path(path)
+            coro = self.loop.create_unix_datagram_server(mock.MagicMock, path)
+            transport, protocol = self.loop.run_until_complete(coro)
+            transport.close()
+            self.loop.run_until_complete(transport.wait_closed())
+            assert protocol.connection_made.call_count == 1
+
+    def test_create_unix_datagram_server_existing_path_nonsock(self):
+        with tempfile.NamedTemporaryFile() as file:
+            coro = self.loop.create_unix_datagram_server(mock.MagicMock,
+                                                         file.name)
+            with self.assertRaisesRegex(OSError,
+                                        'Address.*is already in use'):
```

Comment:
I will check both. So we make sure the error message is also correctly rewritten